NetCore/LoadResizeSave.cs

```diff
@@ -244,51 +246,13 @@ public void MagicScalerResize(string input)
                 Width = ThumbnailSize,
                 Height = ThumbnailSize,
                 ResizeMode = CropScaleMode.Max,
-                SaveFormat = FileFormat.Jpeg,
-                JpegQuality = Quality,
-                JpegSubsampleMode = ChromaSubsampleMode.Subsample420
+                EncoderOptions = new JpegEncoderOptions(Quality, ChromaSubsampleMode.Subsample420, true)
             };
 
-            using (var output = new FileStream(OutputPath(input, MagicScaler), FileMode.Create))
-            {
-                MagicImageProcessor.ProcessImage(input, output, settings);
-            }
-        }
-
-        [Benchmark(Description = "SkiaSharp Canvas Load, Resize, Save"/*,
-            OperationsPerInvoke = ImagesCount*/)]
-        public void SkiaCanvasBenchmark()
-        {
-            foreach (string image in Images)
-            {
-                SkiaCanvasResize(image);
-            }
-        }
-
-        public void SkiaCanvasResize(string input)
```

Comment:
This deleted function is still referenced in `LoadResizeSaveParallel.cs`.
https://github.com/bleroy/core-imaging-playground/blob/5bbfba070a161a3f5c4b77799e6f4a8fce3b6d48/NetCore/LoadResizeSaveParallel.cs#L47